packages/flutter/lib/src/material/calendar_date_picker.dart

```diff
@@ -799,13 +799,19 @@ class _MonthPickerState extends State<_MonthPicker> {
                 children: <Widget>[
                   const Spacer(),
                   IconButton(
-                    icon: const Icon(Icons.chevron_left),
+                    icon: Icon(
+                      Icons.chevron_left,
+                      semanticLabel: _localizations.previousMonthTooltip,
```

Comment:
We should only do this if platform is android. In other platform this will be announce twice